src/backend/users.js

```diff
@@ -0,0 +1,24 @@
+const router = require('express').Router();
+
+router.get('/users', (req, res) => {
+    req.app.get('UsersService').callbackMeTheListing((result, err) => {
+        req.app.get('ErrorHandler').returnResponse(result, err, res);
```

Comment:
`app.set` służy do ustawiania konfiguracji expressa. Wykorzystywanie konfiguracji biblioteki do przekazywania referencji na zależności nie może się dobrze skończyć na dluższą metę.

Chciałeś tu osiągnąć coś na kształt dependency injection.
Wygląda nawet ładnie, ale wystarczyło zrobić `require` na górze tego pliku, albo (polecam) odwrócić relację tego pliku z głównym - eksportować tu funkcję, która przyjmuje referencje na zainicjowane zależności jeśli trzeba i zwraca router. (może nawet przyjmować referencję na router i nic nie zwracać, ale to mniej ładne rozwiązanie)